client/my-sites/patterns/components/pattern-type-switcher/style.scss

```diff
@@ -0,0 +1,37 @@
+.pattern-library__toggle--type {
```

Comment:
Hm, what about?
```suggestion
.pattern-type-switcher {
```

Reply:
At least, I think, we should avoid using modification "--" for the root class of the **independent** component.
WDYT?

Reply:
Yeah, you're probably right that we should address this. This component is still styled by `client/my-sites/patterns/components/pattern-library/style.scss` to some extent. That's bound to be confusing in the future. I'll look into this